OpenOversight/app/commands.py

```diff
@@ -61,11 +61,14 @@ def make_admin_user():
             break
         print("Passwords did not match")
 
+    admin_user = User.query.filter_by(is_administrator=True).first()
```

Comment:
This does presume that there is at least one admin user in the system.

Reply:
If we're using `confirmed_by` for auditing, this technically wouldn't be the user that this new user was confirmed by. Since we don't have the concept of a system user and this field isn't nullable, I set confirmed_by to the user's own id in my change. What do you think?

Reply:
I think that's a better solution, I'm here for it. I'll make the change tomorrow.

Reply:
Done!